src/components/common/YourCompany/EntityName.vue

```diff
@@ -348,7 +378,7 @@ export default class EntityName extends Mixins(NameRequestMixin) {
   }
 
   /** The current options for name changes. */
-  get correctionNameChoices (): Array<CorrectNameOptions> {
```

Comment:
Renamed because this is not exclusive to corrections. I left it as "correct name choices" because the component that implements this is called CorrectName.vue.